### PR TITLE
Test case fix

### PR DESCRIPTION
**Pull request by nukenukenukelol**
Fix the following test cases:

TestRayBeginInsideCircleAndDoesNotGoThroughOrigin
TestNegativeXandY 
TestBeginAtThetaBoundaryInOrthogonal

### CODE REVIEW

**Comment by cgyurgyik** [comment]
Since this is our second try at getting test cases correct, can you put some pictures of the test cases you've corrected so we can confirm the correct voxels are hit?

**Comment by cgyurgyik** [comment]
Also, you have % TODOs somewhere randomly in the test file. Remove that, and at the bottom put this list of TODOs:
```
% TODOs
% - Ray intersects a “theta boundary” outside the circle, and ensure it does 
%   NOT record this as a theta hit.
% - Ray does not go through the innermost voxel.
% - Ray that traverses a single voxel.
% - Ray that ends inside the circle.
% - Ray with positive x- and negative y- directions.
% - Ray with negative x- and positive y- directions.
% - Ray that has a t_begin time different than 0.0
% - Ray that has an odd number of angular sections.
% - Ray that tightly intersects a voxel to determine if floating point tolerances 
%   will still deem it an intersection.
```

**Comment by nukenukenukelol** [comment]
> Since this is our second try at getting test cases correct, can you put some pictures of the test cases you've corrected so we can confirm the correct voxels are hit?
> 
> You also have the following test case:
> 
> ```
> function testRayGoesAlongRayBoundary(testCase)
> ```
> 
> What does this mean and/or test?

This is the corner test that if a ray is going along one of the boundaries.
Screen Shot 2020-02-29 at 11 20 50 AM

**Comment by cgyurgyik** [comment]
> > Since this is our second try at getting test cases correct, can you put some pictures of the test cases you've corrected so we can confirm the correct voxels are hit?
> > You also have the following test case:
> > ```
> > function testRayGoesAlongRayBoundary(testCase)
> > ```
> > 
> > 
> > What does this mean and/or test?
> 
> This is the corner test that if a ray is going along one of the boundaries.
> Screen Shot 2020-02-29 at 11 20 50 AM

Ok, maybe then we can rename it to along theta boundary or something similar. That is not the ray boundary.

File: polarCoordinateTraversalTest.m [hunk 1]
```diff
@@ -158,7 +158,7 @@ function testNegativeXandY(testCase)
     
     [rVoxels, thetaVoxels] = polarCoordinateTraversal(min_bound, max_bound, ray_origin, ray_direction, ...
         circle_center, circle_max_radius, num_radial_sections, num_angular_sections, t_begin, t_end, verbose);
-        expected_rVoxels     = [1,2,2,3,3];
+        expected_rVoxels     = [1,2,2,1,1];
```
Comment (cgyurgyik) [inline polarCoordinateTraversalTest.m]: ```expected_rVoxels = [1,1,2,2,1]```

**Pull request merged**